loopdb/sqlc/migrations/000004_is_wallet_addr.up.sql

```diff
@@ -0,0 +1,3 @@
+-- is_external_addr indicates whether the destination address of the swap is not
+-- a wallet address.
+ALTER TABLE loopout_swaps ADD is_external_addr BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
IIUC this assumption us incorrect as there might be multiple ongoing swaps when the user updates that may or may not have a dest addr set to an external address.